lms/envs/common.py

```diff
@@ -2230,15 +2230,6 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 
 ################################# CELERY ######################################
 
-# Celery's task autodiscovery won't find tasks nested in a tasks package.
-# Tasks are only registered when the module they are defined in is imported.
-CELERY_IMPORTS = (
-    'poll.tasks',
```

Comment:
Hey @feanil , edX has gotten reports that XBlock-poll's reporting features are no longer working ([TNL-8370](https://openedx.atlassian.net/browse/TNL-8370)), and I think it was caused by this change (but went unnoticed for months?).

The issue is that the `poll` XBlock is not a django app, so its `tasks.py` won't get auto-discovered by celery. If so, is it OK if I fix it with a PR here to reinstate `CELERY_IMPORTS = ('poll.tasks', )`, or is there some other way I should be doing that now?

Previous PR: https://github.com/edx/edx-platform/pull/23700

Reply:
Hey Braden, that seems like a fine fix, the other option would be to make the polls code into a django app? Which I would prefer but either is acceptable.